NGitLab.Tests/Docker/GitLabDockerContainer.cs

```diff
@@ -264,8 +283,10 @@ private async Task SpawnDockerContainerAsync()
         TestContext.Progress.WriteLine("GitLab Docker container is ready");
     }
 
-    private async Task GenerateCredentialsAsync()
+    private async Task GenerateCredentialsAsync(IBrowserContext context)
```

Comment:
We have way too many context types... Do you mind renaming the parameter to `browserContext`?